src/applications/vaos/tests/new-appointment/components/DateTimeSelectPage/index.unit.spec.jsx

```diff
@@ -343,4 +345,321 @@ describe('VAOS <DateTimeSelectPage>', () => {
     // Cleanup
     resetFetch();
   });
+
+  it('should show validation error if no date selected', async () => {
```

Comment:
Noticed this validation state wasn't covered when looking at code coverage report